### PR TITLE
Refactor app to use syndccutils functions

### LINKED ISSUE #25: Extract manifest generating code into a separate package
@teslajoy and I have discussed the idea of extracting some of the code from the shiny app into a separate package, which would give us the option of creating manifests programmatically in addition to through the UI. The annotationUI app could then use functions from the package, and all user-facing functionality would remain the same.

I talked with @jaeddy and we think this is a good fit for [syndccutils](https://github.com/Sage-Bionetworks/syndccutils/). I can work on a PR to that repo first, and once the functions are added update annotationUI.

### PR DESCRIPTION
**Pull request by karawoo**
Closes #25. I've refactored the shiny app to load the `syndccutils` package and use the functions in it to generate manifests. There was some redundant loading of packages happening as well (`require()` followed by `library()`), so I removed some of the duplication. Now packages are loaded with `library()` only.

I've tested this out locally and believe it works fine. @kdaily or @jaeddy would you mind taking a look?

Also, I'm not familiar with our shiny server setup -- is someone able to give me a quick walkthrough of how to deploy this once it's approved?

### CODE REVIEW

**Comment by jaeddy** [comment]
Cool! I'll take a look.

I don't know about our shiny server — @kdaily or @xindiguo would probably be able to advise.

File: global.R [hunk 1]
```diff
@@ -2,38 +2,32 @@
...
-{
-  if (!is.element(p, installed.packages()[,1]))
-    install.packages(p, dep = TRUE)
-  require(p, character.only = TRUE)
+usePackage <- function(p, github, repos = getOption("repos"), ...) {
+  if (!missing(github)) {
+    ## Install from github -- devtools::install_github will automatically skip
+    ## if SHA of most recent commit matches what's installed already
+    devtools::install_github(paste(github, p, sep = "/"), ...)
+  }
```
Comment (jaeddy) [inline global.R]: This doesn't seem to be the case in practice: `usePackage()` re-installs `syndccutils` (and all of its dependencies... from source) every time I call it locally. :( I have `devtools` v1.13.4 installed.

Comment (karawoo) [reply global.R]: What version of git2r do you have installed? I think there was an SSL issue in that package that resulted in `install_github()` always reinstalling. Can you try updating to the current github version of git2r? If that solves it I an update the DESCRIPTION and make sure it gets the right version.

Comment (jaeddy) [reply global.R]: I upgraded to v0.21.0 of *git2r*, but that didn't seem to fix the problem. Do you recommend installing the dev version from GitHub (just realized how meta that is...)?

Comment (karawoo) [reply global.R]: Yeah that's what I'd recommend

**Pull request merged**